### PR TITLE
Waive copyright under CC0

### LINKED ISSUE #1: Add Creative Commons "license"
probably https://creativecommons.org/share-your-work/public-domain/cc0

### PR DESCRIPTION
**Pull request by nickrobinson251**
closes #1 

@omus should probably adjudicate if this is what we want (as opposed to an OSI license which is for software)

### CODE REVIEW

**Comment by omus** [comment]
I'll need to review the license yet. I do believe that we should probably have an explicit license file according to: https://help.github.com/en/articles/licensing-a-repository

**Comment by nickrobinson251** [comment]
Added a LICENSE file

This matches e.g. how github/gitignore is licensed https://github.com/github/gitignore
You can see other examples by searching GitHub for `license:cc0-1.0`

File: README.md [hunk 1]
```diff
@@ -977,3 +980,8 @@ To change it:
 1. Go to `Atom -> Preferences -> Packages`.
 2. Search for the "language-julia" package and open the settings for it.
 3. Find preferred line length (under "Julia Grammar") and change it to 92.
+
+## Copyright waiver
+
+This work is in the public domain.
+Copyright and related rights are waived via [CC0](https://creativecommons.org/publicdomain/zero/1.0/).
```
Comment (omus) [inline README.md]: Seems redundant now. Should we just have the LICENSE file?

**Comment by omus** [comment]
I'm good with the CC0 license

File: README.md (hunk 1 above)
Comment (nickrobinson251) [reply README.md]: yeah, that seems fair enough 

I liked linking to the "human-readable" version, but we just use a LICENSE file everywhere else :) Will change

Comment (nickrobinson251) [reply README.md]: done

**Pull request merged**